src/latexify/core.py

```diff
@@ -135,6 +150,8 @@ def visit_Attribute(self, node, action):  # pylint: disable=invalid-name
 
     def visit_Name(self, node, action):  # pylint: disable=invalid-name
         del action
+        if node.id in self.assign_var.keys():
+            return self.assign_var[node.id]
```

Comment:
To avoid exponential expansion, I suggest to add an option to activate or not the assignment.
For example:
```python
@latexify.with_latex(assign_mode=False)
def h(x):
    a = abs(x)
    b = math.exp(math.sqrt(x))
    return a * b
```
will return `a = \left|{x}\right| \\ b = \exp{\left({\sqrt{x}}\right)} \\ \mathrm{h}(x) \triangleq ab`

![image](https://user-images.githubusercontent.com/95867256/195809394-e86c0070-ab00-47bc-842d-319376755bbb.png)

Reply:
There is also a problem when assignment refers to several numbers with a product like:
```python
@with_latex
def f(x):
    a = 1
    b = 2
    return a * b
```
return `\mathrm{f}(x) \triangleq 12`

![image](https://user-images.githubusercontent.com/95867256/195810524-49e609e1-6a5b-4cf0-a181-97fb402ebf90.png)